### PR TITLE
Jira: Mark as FIPS Compliant in app metadata

### PR DESCRIPTION
**Pull request by shiyingtu**
- Add fips_compliant key and set value to true
- Fix bad Copyright text

Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
- [x] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by paulnawat** [comment]
@shiyingtu  From the description, we also want to revert the phantom version here, right?

File: jira_get_ticket.html [hunk 1]
```diff
@@ -9,9 +9,17 @@
 {% block widget_content %} <!-- Main Start Block -->
 
 <!-- File: jira_get_ticket.html
-  Copyright (c) 2016-2021 Splunk Inc.
+  Copyright (c) 2019-2021 Splunk Inc.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
```
Comment (paulnawat) [inline jira_get_ticket.html:15]: Are these document changes intentional?

Comment (gary-phantom) [reply jira_get_ticket.html:15]: why does copyright year changed from 2016 to 2019?

Comment (shiyingtu) [reply jira_get_ticket.html]: These document changes are intentional to solve failed static test: "Copyright". [Fix commit](https://github.com/splunk-soar-connectors/jira/pull/1/commits/0904f748e1f2be65d2b5f847d559b911e82d932a) already included.

**Review by rgil-splunk — APPROVED** [review]

**Pull request merged**